packages/web-worker/src/test/e2e.test.ts

```diff
@@ -0,0 +1,138 @@
+/**
+ * @jest-environment node
+ */
+
+import * as path from 'path';
+import {WebWorkerPlugin} from '../webpack-parts';
+import {withContext, Context, runWebpack as runWebpackBase} from './utilities';
```

Comment:
Some really nice testing utilities here! Maybe this could be its own package one-day.